src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -560,6 +577,118 @@ column_type:
 | char_type
 | time_type
 
+column_option_list_opt:
+  {
+    $$.ColOptList = []*ColumnOption{}
+  }
+| column_option_list
+  {
+    $$.ColOptList = $1
+  }
+
+column_option_list:
+column_option
+  {
+    $$ = append($$, $1)
+  }
+| column_option_list column_option
+  {
+    $$ = append($1, $2)
+  }
+
+column_option:
```

Comment:
optimize the code

Reply:
OK, I`ll make the assignment code  more consise.

Reply:
> optimize the code

I had optmize the code here and push update the code, please review again.